lib/systems/parse.nix

```diff
@@ -520,7 +520,7 @@ rec {
         cpu = cpu.name;
         vendor = vendor.name;
         kernel = kernelName kernel;
-        inherit optExecFormat;
+        #inherit optExecFormat;
```

Comment:
Why doesn't this commit remove the line?

Reply:
Thanks! 62aaddf904bd